lib/design_logic/star.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:parabeac_core/design_logic/design_shape.dart';
+
+// part 'star.g.dart';
+
+// @JsonSerializable()
+abstract class Star extends DesignShape {
+  Star();
+  // factory Star.fromJson(Map<String, dynamic> json) => _$StarFromJson(json);
```

Comment:
Lets remove the comments since we concluded that these classes are not going to be JsonSerializable